### ISSUE TITLE
Using parallel coordinates w/ plotly_dark theme causes y tick label artifacts

### ISSUE THREAD

**Issue by adamtec** [plotly/plotly.py#2643]
Using parallel coordinates w/ plotly_dark theme causes y tick label artifacts. Using the basic example provided in the docs and setting template to plotly_dark can be used to reproduce. Looks like there are multiple sets of tick labels overlapping. It's not noticeable on templates with white backgrounds.

```
import plotly.express as px
df = px.data.iris()

fig = px.parallel_coordinates(df, color="species_id", template='plotly_dark', labels={"species_id": "Species",
                "sepal_width": "Sepal Width", "sepal_length": "Sepal Length",
                "petal_width": "Petal Width", "petal_length": "Petal Length", },
                             color_continuous_scale=px.colors.diverging.Tealrose,
                             color_continuous_midpoint=2)
fig.show()
```
![Screen Shot 2020-07-16 at 8 51 46 AM](https://user-images.githubusercontent.com/35566983/87672893-aa2b7d80-c741-11ea-83dd-b4f768ca3a18.png)

**Comment by adamtec** [comment]
Any update? Any dark based theme template results in illegible text. I think some white text-shadow is being used regardless and I don't see any knobs to change it.

**Comment by nicolaskruchten** [comment]
That's right, we've got a fix in the `master` branch of Plotly.js but we need a few more weeks before those changes ripple down to Plotly.py.

**Comment by nicolaskruchten** [comment]
This is fixed in version 5.0 of `plotly`.
